### PR TITLE
:bug: Source MySQL CDC: sync failed when has Zero-date value in mandatory column

### LINKED ISSUE #6884: Source MySQL CDC: sync failed when has null value in `created_at` column
## Enviroment
Is this your first time deploying Airbyte: No
OS Version / Instance:  c5.4xlarge
Memory / Disk:  (16vCPU and 32GB ram)
Deployment: AWS Kubernetes
 Airbyte Version: 0.30.1-alpha
Step: Initial Sync
Description: We are trying to replicate from Aurora RDS MySQL Reporting Replica with Binary Logs (CDC) to AWS S3, and we have some tables that have datetime fields that either have 0 or null values.  We run into Java Heap errors like the following:

[Slack convo](https://airbytehq.slack.com/archives/C01MFR03D5W/p1633045225041800)

## Current Behavior
image

## Expected Behavior
*Tell us what should happen.*

## Logs
[logs-165-0 masked.txt](https://github.com/airbytehq/airbyte/files/7307893/logs-165-0.masked.txt)

## Steps to Reproduce
1.
2.
3.

## Are you willing to submit a PR?

Remove this with your answer.

### PR DESCRIPTION
**Pull request by DoNotPanicUA**
## What
An incremental sync fails if not null column has a Zero-date value. The Debezium driver should replace it with the epoch but it left null as for optional.

## How
We already set default values for NULL optional values. Now we also analyze if it's a date and still null, put epoch for a mandatory column.

### CODE REVIEW

File: airbyte-integrations/connectors/source-mysql/Dockerfile [hunk 1]
```diff
@@ -16,5 +16,5 @@ ENV APPLICATION source-mysql
 
 COPY --from=build /airbyte /airbyte
 
-LABEL io.airbyte.version=0.5.2
+LABEL io.airbyte.version=0.5.3
```
Comment (mkhokh-33) [inline airbyte-integrations/connectors/source-mysql/Dockerfile:19]: should we update source-mysql-strict-encrypt as well?

**Review by mkhokh-33 — APPROVED** [review]

**Review by alexandr-shegeda — APPROVED** [review]
LGTM

**Comment by alexandr-shegeda** [comment]
@DoNotPanicUA we can proceed with publishing and merging without airbyte review

**Pull request merged**